.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: required
 language: cpp
 dist: trusty
+branches:
```

Comment:
It can be removed now; that was a branch that was only for documentation updates, thus not needing travis testing.